audit/audit.go

```diff
@@ -0,0 +1,108 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package audit
```

Comment:
I was wondering, what are your thoughts on moving this to the service layer ? This seems like a better fit there, and doesn't need to be a top-level domain object.

Reply:
The cloud team asked if I could create a package they could use in their code as well.  Once I add the config processing (phase 2) I plan to break this out into a package more easily shared.  I won't be able to do that if I embed it in the service layer.  Also, I'll be using this to audit the CLI in a separate PR.  I do see your point about adding a relatively small package to the top level, but it won't be there very long.

Reply:
Sounds good. Do you plan to have a separate repo for this ? Or move this to somewhere else ?

Reply:
I think it will end up in its own mattermost repo once the config processor and some built-in targets are added, unless someone has a better idea.